<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-Q4DCNU94" failures="0" tests="4" name="fita.tests.Test001" time="8.446" errors="1" timestamp="2024-09-11T09:32:01 IST" skipped="2">
  <testcase classname="fita.tests.Test001" name="addProduct" time="0.000">
    <skipped/>
  </testcase> <!-- addProduct -->
  <testcase classname="fita.tests.Test001" name="checkIfProductAdded" time="0.000">
    <skipped/>
  </testcase> <!-- checkIfProductAdded -->
  <testcase classname="fita.tests.Test001" name="loginApp" time="1.178">
    <error message="Fail to save: an error occurs while saving the package : The part /xl/theme/theme1.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4acbf86f. Enable logging via Log4j 2 for more details." type="org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException">
      <![CDATA[org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /xl/theme/theme1.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4acbf86f. Enable logging via Log4j 2 for more details.
at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:643)
at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1532)
at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1519)
at org.apache.poi.openxml4j.opc.ZipPackage.closeImpl(ZipPackage.java:515)
at org.apache.poi.openxml4j.opc.OPCPackage.close(OPCPackage.java:502)
at org.apache.poi.ooxml.POIXMLDocument.close(POIXMLDocument.java:189)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.close(XSSFWorkbook.java:640)
at fita.utilites.Utilities.getPassword(Utilities.java:49)
at fita.pageobjects.LoginPage.enterUserpassword(LoginPage.java:24)
at fita.pageobjects.LoginPage.performLogin(LoginPage.java:35)
at fita.tests.Test001.loginApp(Test001.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /xl/theme/theme1.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4acbf86f. Enable logging via Log4j 2 for more details.
at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:632)
... 25 more
]]>
    </error>
  </testcase> <!-- loginApp -->
  <testcase classname="fita.tests.Test001" name="openUrl" time="7.268"/>
</testsuite> <!-- fita.tests.Test001 -->
